pkg/operator/ceph/csi/csi.go

```diff
@@ -274,11 +274,11 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 			CSIParam.ProvisionerReplicas = 1
 		} else {
 			replicas := k8sutil.GetValue(r.opConfig.Parameters, "CSI_PROVISIONER_REPLICAS", "2")
-			r, err := strconv.ParseInt(replicas, 10, 32)
+			rp, err := strconv.ParseInt(replicas, 10, 32)
```

Comment:
How about naming the vars `replicas --> replicaStr` and `rp --> replicas`? `rp` isn't a readable var name IMO.